plotter/db.py

```diff
@@ -774,10 +861,12 @@ class Models(Resource):
 
 
 class Breakpoints(Regions):
+    keys = ("user", "name", "chrom")
```

Comment:
now I define `keys` for both `Breakpoints` and `Copies`, so they will be added to `DB_CLASSES` and so that their metaclass will open a database for them